utils/swift_build_support/swift_build_support/products/wasmstdlib.py

```diff
@@ -194,3 +197,94 @@ def add_extra_cmake_options(self):
                                   '-Xcc;-mthread-model;-Xcc;posix;'
                                   '-Xcc;-pthread;-Xcc;-ftls-model=local-exec')
         self.cmake_options.define('SWIFT_ENABLE_WASI_THREADS:BOOL', 'TRUE')
+
+
+class WasmSwiftSDK(product.Product):
```

Comment:
Since this is WASI-specific, would it make sense to reflect that in the product name?

```suggestion
class WASISwiftSDK(product.Product):
```

Reply:
This SDK would eventually support multiple targets within the single SDK, so it's not WASI spesific.